docs/articles/documentation/recipes/testing-select-elements.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: docs
+title: Testing %lt;select&gt; Elements
+permalink: /documentation/recipes/testing-select-elements.html
+---
+# Testing \<select\> Elements
+
+This recipe shows how to test `<select>` elements and pick options from the drop-down list.
+
+Assume the following `<select>` element.
+
+```html
+<select name="city">
```

Comment:
`city` is name attribute but you [use it](https://github.com/DevExpress/testcafe/pull/1586/files#diff-cd94f71b09ce72ec3b745dce1ea5febcR44) like `id` and test will not work
I think we should run our examples on simple pages to avoid similar things